### ISSUE TITLE
 [35]ERROR |  || Couldn't find any scrapable media at all!

### ISSUE THREAD

**Issue by datguytry15** [prof79/fansly-downloader-ng#36]
WARNING | 14:23 || Could not find a chat history with shybabybun; skipping messages download ...
 Info | 14:23 || Executing Timeline functionality. Anticipate remarkable outcomes!

 Info | 14:23 || Inspecting most recent Timeline cursor ... [CID: 352820826220802048]
 [35]ERROR | 14:23 || Couldn't find any scrapable media at all!
 This most likely happend because you're not following the creator, your authorisation token is wrong
 or the creator is not providing unlocked content.

An error began to appear, I changed the token, nothing changed.

**Comment by oksndr** [comment]
[ Info ] [2024-03-16 | 03:49]: Executing Timeline functionality. Anticipate remarkable outcomes!
[ Info ] [2024-03-16 | 03:49]: Inspecting most recent Timeline cursor ... [CID: 284853427224195072]
[ [35]ERROR ] [2024-03-16 | 03:49]: Couldn't find any scrapable media at all!                
 This most likely happend because you're not following the creator, your authorisation token is wrong                
 or the creator is not providing unlocked content.

Same issue, yesterday morning it was working fine, nothing changed (token or user agent) and now it won't download stuff from timeline, content at messages is being downloaded without problems.

**Comment by Trouble03** [comment]
Same here, don't know what to do

**Comment by prof79** [comment]
See #34

**Comment by prof79** [comment]
Have fun ^^

https://github.com/prof79/fansly-downloader-ng/releases/tag/v0.9.1

**Comment by Trouble03** [comment]
Did a fresh install, still got this :

 Info | 10:15 || Executing Timeline functionality. Anticipate remarkable outcomes!

 Info | 10:15 || Inspecting most recent Timeline cursor ... [CID: 227317562784874496]
 [35]ERROR | 10:15 || Couldn't find any scrapable media at all!
 This most likely happend because you're not following the creator, your authorisation token is wrong
 or the creator is not providing unlocked content.

**Comment by FMon77** [comment]
I also get the above with version 0.9.1. I did test with different target users and tried a fresh new config creation. I'm using Firefox as the browser. 
Thank you very much for your hard work!

**Comment by datguytry15** [comment]
> I also get the above with version 0.9.1. I did test with different target users and tried a fresh new config creation. I'm using Firefox as the browser. Thank you very much for your hard work!

same here with 0.9.1

**Comment by prof79** [comment]
Maybe they have changed something again - and since they seem to now also terminate accounts using the downloader maybe I should just give up on this. My work is tempting enough and I do not have the time nor fancy to play daily cat-and-mouse with the Fansly security team in my spare time and I also do not have the resources to reverse and code an exact replica of the original web app. I'm a paying customer for some creators and I am not a pirate, just want to use/browse my lib on VR goggles - I guess they totally lack imagination of use cases.

**Comment by FMon77** [comment]
Totally understand. Who knows how many people you're up against on their team.  I do appreciate the time and dedication you did put in it. I managed to use it for a little bit.
I'm sort of on the same lines. I do pay for a few and would like to have a copy of what I paid for to view later, I never share since they own the content. I managed to skim by the bans because I've only tested here and there so it wouldn't hit too many times and the bot auto ban me.
It sucks that they are really trying to lock it down. Backup plan is screen record/snip unless someone does find a way to cut into their security and make it much harder on them.
Thanks again!
